### DISCUSSION TITLE
v5.0.0

### DISCUSSION THREAD

**Discussion by SteffeyDev** [danielbuechele/atemOSC#281] [Announcements]
atemOSC is now fully supported and back under active development! I've rebuilt the app with a new UI, new features, and better documentation. I've decided to move away from GitHub as a distribution & support platform to make things simpler for many of you who are not software developers. Go to [atemosc.com](https://atemosc.com) to see the future of atemOSC and download v5.

Version 5 is a complete rebuild on a new tech stack that will allow me to port AtemOSC to Windows and Linux relatively easily. My goal was to be fully backwards compatible with Version 4 and earlier, with a few exceptions:
- Hyperdeck control has been completely re-worked. AtemOSC now connects directly to HyperDecks and HyperDecks are first-class objects in AtemOSC, just like switchers (resolves #58). You can now use `/hyperdeck/<nickname>/<command>` to control HyperDecks, similar to how you use `/atem/<nickname>/<command>` to control switchers. There's also a host of new commands available for HyperDecks, including the ability to set the recording filename.
- Re-implementing some of the commands proved more challenging, so I decided I'll wait to do the work until someone needs them. Let me know if you need any of these: `/usk/<key>/fly/reset`, `/usk/<key>/fly/clear`, `/usk/<key>/mask/reset`, `/supersource/box/<key>/crop-reset`, `/dsk/<key>/mask/reset`.

One major change with this version is that atemOSC is no longer free or open source. This was a necessary change for the survival of the project; I've reached a point in life where I either had to find a way to get a return on investment for the time I spent maintaining atemOSC, or I'd have to archive it and leave all of you hanging. A while ago, I put out a request for others to help maintain the project and keep it open source, but no one stepped up. I decided that the best path forward is to sell software licenses for atemOSC so that I can justify continuing to work on it, and take time out of my workday to fix bugs, add features, and be available to support you.

I'm very excited for this opportunity and hope that you will support my past and future development of atemOSC by buying a license. I'm looking forward to being able to spend more time on atemOSC, instead of just working on it very occasionally on weekends, and I believe this model will allow me to serve you better.

<hr /><em>This discussion was created from the release <a href='https://github.com/SteffeyDev/atemOSC/releases/tag/v5.0.0'>v5.0.0</a>.</em>

**Comment by mark-morreau** [comment]
Yes, I would buy a licence for a Windows version, assuming it's not crazy expensive :-) .  You don't put the pricing on the website, do you?
If it's on a per-machine basis I would buy a couple.
